Documents/MobileAppActors.txt

```diff
@@ -0,0 +1,17 @@
+Actors: Regular User, Artist, Admin
+
+Regular User - Draw on Digital Prosthetic, Browse Gallery, Contact Artists
+A regular user would use this app for fun, to enhance their creativity by drawing on the flattened surface of a prosthetic, or simply to browse the galleries without actually visiting the website.
```

Comment:
**Browse gallery**- describe what artwork does the user see. Is it all artwork from the app, or just the approved one? What images end up at the top?
**Like**- would it support liking?
**Filter**- can you filter?